specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2022-10-01-preview/account.json

```diff
@@ -0,0 +1,1764 @@
+            "$ref": "#/definitions/Rule"
+          }
+        },
+        "storeType": {
+          "type": "string"
+        }
+      }
+    },
+    "FastRegex": {
+      "type": "object",
+      "properties": {
+        "maxDigits": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "maxLetters": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minDashes": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minDigits": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minDigitsOrLetters": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minDots": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minHex": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minLetters": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "minUnderscores": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "options": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "regexStr": {
+          "type": "string"
+        }
+      }
+    },
+    "Rule": {
+      "required": [
+        "qualifiedName"
+      ],
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "isResourceSet": {
+          "default": true,
+          "type": "boolean"
+        },
+        "lastUpdatedTimestamp": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "qualifiedName": {
+          "type": "string"
+        }
+      }
+    },
+    "ResourceSetRuleConfigList": {
+      "description": "Paged list of account resources",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "count": {
+          "format": "int64",
+          "description": "Total item count.",
+          "type": "integer"
+        },
+        "nextLink": {
+          "description": "The Url of next result page.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Collection of items of type results.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceSetRuleConfig"
+          }
+        }
+      }
+    },
+    "AccountPreferences": {
+      "description": "Account Preferences resource.",
+      "type": "object",
+      "properties": {
+        "cobranding": {
+          "$ref": "#/definitions/Cobranding"
+        }
+      }
+    },
+    "Cobranding": {
+      "description": "Cobranding object.",
+      "type": "object",
+      "properties": {
+        "logo": {
+          "description": "logo",
+          "type": "string"
+        },
+        "color": {
+          "description": "color",
```

Comment:
I would expect more detail about what is expected. I saw "blue" in example, does it accept e.g. RGB#, or "green yellow"?

Reply:
Added description and changed example accordingly